build.gradle

```diff
@@ -444,8 +444,7 @@ patchPluginXml {
   sinceBuild intellijVersionToBuildNumber(intellijVersions.earliestSupported)
 
   // In `untilBuild` situation is inverted: it's inclusive when using `*` but exclusive when without `*`
-  untilBuild intellijVersionToBuildNumber(intellijVersions.current) + '.*'
-
+  untilBuild '202.*'
```

Comment:
Is it intentionally checked in? If that's just meant for local development/testing against EAPs, consider making it configurable with an env var

Reply:
whooops